### ISSUE TITLE
Stuff

### ISSUE THREAD

**Issue by trebor57** [dev-osrose/osIROSE-new#199]
I don't know what the progress of the server is, so just the little things I notice

1 - new char appears in zant
2 - MP seems to go down on its own
3 - mobs don't attack
4 - when I hit anything, it says Miss and how much I hit with (Red and blue)
5 - can't place skills on bar
6 - says I have a new quest, but nothing in quest box
7 - nothing showing how much exp I am getting
8 - How do I change the location of where my files are located, I didn't get it the first time, so they all are in a root directory.  I uninstalled and reinstalled, but it placed them all the in same place.

Like I said, I don't know what the details are on where everything is along on this version  :)

**Comment by RavenX8** [comment]
1) it's in zant because quests aren't implemented yet so you would never be able to leave birth island if we placed you there.
2) MP and HP isn't being adjusted when leveling up, so after a level up, the server updates the client with the current HP and MP.
3) Mob AI isn't implemented yet
4) This was something that I had fixed but it was broken again with the latest update. Looking into this
5) Not implemented. See @L3nn0x 
6) Not implemented
7) You exp is in the top left under MP
8) What files are you talking about?

**Comment by trebor57** [comment]
The launcher files is what I am talking about.   When I ran the setup    OsIROSE Test Server Setup 0.1.1.exe    It said something about the setup, I must have clicked wrong, so it places the trose.exe and other files into a spot I don't want them.   So, there must be a config that points to that location someplace?

**Comment by RavenX8** [comment]
yeah it's in `AppData/Roaming/osirose-updater/config.json` I will also add a launch option to clear out the config file

**Comment by trebor57** [comment]
that was it, thank you.  good idea or when you uninstall the program which I did that it remove that set of directories

**Comment by RavenX8** [comment]
Alright `--first-start` flag added to the launcher and it has been set to remove the config files on uninstall
